static/scripts/install-nim-bundle.sh

```diff
@@ -519,6 +532,24 @@ printUsageInfo(){
   exit 0
 }
 
+printSupportedOS(){
+  echo "This script can be run on the following operating systems
+       [-d distribution (ubuntu20.04,ubuntu22.04,ubuntu24.04,debian11,debian12,centos8,rhel8,rhel9,oracle7,oracle8,amzn2)] [-h print help]"
+  printf "\n  ubuntu20.04(focal)"
+  printf "\n  ubuntu22.04(jammy)"
+  printf "\n  ubuntu24.04(noble)"
+  printf "\n  debian11"
+  printf "\n  debian12"
```

Comment:
Since we're including the code names for the Ubuntu distros, I suggest:

```suggestion
  printf "\n  debian11(bullseye)"
  printf "\n  debian12(bookworm)"
```